packages/shikiji/package.json

```diff
@@ -26,6 +26,8 @@
       "default": "./dist/core.mjs"
     },
     "./wasm": {
+      "unwasm": "./dist/onig.wasm",
+      "wasm": "./dist/onig.wasm",
```

Comment:
Why are we having additional `wasm` conditions? I mainly wanted to avoid using a condition that looks standard but is not yet, an explicit condition (similar to runtime-key conditions)